src/community/taskmanager/src/main/java/org/geoserver/taskmanager/external/DefaultDialectImpl.java

```diff
@@ -0,0 +1,59 @@
+/* (c) 2017 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.taskmanager.external;
+
+
+import java.sql.Connection;
+import java.util.Collections;
+import java.util.Set;
+
+import org.geoserver.taskmanager.util.SqlUtil;
+
+/**
+ * Default implementation for the Dialect interface.
+ * 
+ * This should work with most databases, but it also limits the functionality of the task manager.
+ * 
+ * @author Timothy De Bock
```

Comment:
Are you contributing on behalf of this author?

Reply:
yes